packages/boxel/addon/components/boxel/action-container/usage.gts

```diff
@@ -57,6 +58,13 @@ export default class extends Component {
           @optional={{true}}
           @description="Unstyled area for custom card content and fields; yields a Section component that takes an optional title and applies an appropriate inset"
         />
+        <Args.Bool
+          @name="hideLock"
+          @description="When true, suppress the 'Actions only visible to you' message and lock icon in the chin."
+          @defaultValue={{false}}
+          @value={{this.hideLock}}
+          @onInput={{fn (mut this.hideLock)}}
+        />
```

Comment:
We discussed removing this entirely, right?

Reply:
Merge problem. Will remove.